### PR TITLE
Spelling Fix

### PR DESCRIPTION
**Pull request by emm3ttarmstrong**
Small spelling fixes and README.md changes.

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: learn/api-docs/ballerina/kafka/clients/Consumer.html [hunk 1]
```diff
@@ -754,7 +754,7 @@ <h2>Remote Methods</h2>
                                 <tr>
                                     <td class="module-title truncate functions" title="getSubscription"><a
                                             class="functions" href="#getSubscription">getSubscription</a></td>
-                                    <td class="module-desc"><p>Returns set of topics wich are currently subscribed by the consumer.</p>
+                                    <td class="module-desc"><p>Returns set of topics which are currently subscribed by the consumer.</p>
```
Comment (pubudu91) [inline learn/api-docs/ballerina/kafka/clients/Consumer.html]: Thanks for taking the time and interest to send in a fix for this! However, the API docs are generated from the doc comments in the source code itself. So the actual fix should be done in the following file: https://github.com/ballerina-platform/ballerina-lang/blob/master/stdlib/messaging/kafka/src/main/ballerina/src/kafka/consumer.bal

Can you please send resend the fix to the `ballerina-1.0.x` branch in the https://github.com/ballerina-platform/ballerina-lang repo?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# About Ballerina.io
+Ballerina is an open source programming language and platform for cloud-era application programmers to easily write software that just works.
```
Comment (praneesha) [inline README.md]:
```suggestion
Ballerina is an open source programming language and a platform for cloud-era application programmers to easily write software.
```

I feel "that just works" part is redundant.

File: learn/api-docs/ballerina/kafka/clients/Consumer.html [hunk 3]
```diff
@@ -1546,7 +1546,7 @@ <h2> getSubscription </h2>
         </span>
         
     </div>
-    <p><p>Returns set of topics wich are currently subscribed by the consumer.</p>
+    <p><p>Returns set of topics which are currently subscribed by the consumer.</p>
```
Comment (praneesha) [inline learn/api-docs/ballerina/kafka/clients/Consumer.html]:
```suggestion
    <p><p>Returns a set of topics, which are currently subscribed by the consumer.</p>
```

**Comment by anupama-pathirage** [comment]
@emm3ttarmstrong Thanks for taking the time to fix the issues. But as @pubudu91 suggested already, these are auto-generated files and they should be fixed in the relevant ballerina file [1].

[1] https://github.com/ballerina-platform/ballerina-lang/blob/master/stdlib/messaging/kafka/src/main/ballerina/src/kafka/consumer.bal#L283

Created an issue [2] to track this and @emm3ttarmstrong / @praneesha Can you please send the proper fix along with the suggestions made in the PR.

[2] https://github.com/ballerina-platform/ballerina-lang/issues/20433

**Pull request closed without merging**